perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_host.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Steps to bootstrap with gpu docker
+# Install docker
+
+echo "Installing linux utility packages, like, lsb-release, curl..."
+sudo apt-get update
+sudo apt-get install -y \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+echo "Installing docker framework..."
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+
+echo "Installing driver..."
+sudo apt update && sudo apt install -y build-essential
+BASE_URL=https://us.download.nvidia.com/tesla
+DRIVER_VERSION=450.172.01
+sudo curl -fSsl -O $BASE_URL/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
+
+sudo sh NVIDIA-Linux-x86_64-$DRIVER_VERSION.run -s
```

Comment:
Question: Does this command take care of choosing appropriate options in the prompt that comes up ? If yes, does the '-s' option at the end takes care of it ?

Reply:
Yes, we choose appropriate option by default. 
One of the option we chose - Install NVIDIA's 32-bit compatibility libraries? as yes - which seems fine. Other than that we only press ok for warning and installation completion.